WcaOnRails/app/views/persons/_championship_podiums.html.erb

```diff
@@ -0,0 +1,5 @@
+<% @championship_podiums.each do |championship_type, results| %>
+  <% if results.any? %>
+    <%= render "regional_championship_podiums", title: t("persons.show.championship_podium_types.#{championship_type}"), results: results %>
```

Comment:
This will result in titles like "Continental Championship Podiums" and "National Championship Podiums", right? I think it would be pretty cool if we could instead generate friendly titles like "European Championship Podiums" and "US National Championship Podiums".

In fact, if you look at our [design doc](https://docs.google.com/document/d/1tgIfNxJqYni_PFsMt-hpDSh1YpfoCseLORvLX6d7-l4/edit), you'll see that a `championship_type` is defined as follows:

> A "championship type" would be a string that is one of the following:
> - "world"
> - "greater_china"
> - A continent id from our continents table (see database seed)
> - A country ISO2 code (see database seed)

As I understand this code, your definition of `championship_type` is a bit different, it's either `world`, `continental`, `national`, or `greater_china`.

I think it would make sense to redefine the `championship_podiums` method so it returns things structured by championship_type as defined in the doc, rather than this other kind of championship_type, which I believe would have a side effect of generating more friendly titles.

Reply:
This would be nice! One tricky thing is the case we've just discussed, if someone changes nationality from China to US, then it becomes difficult to show both of his championship title sections (applies to both continental and national titles). Also following your idea would require translators to translate this for every single country, so it's nice but would require quite a lot of effort and I worry it may be an overkill. Thoughts?

Reply:
> then it becomes difficult to show both of his championship title sections (applies to both continental and national titles)

Sorry, I don't think I understand the problem here. Can't each thing that this person was eligible for (and won a title in!) show up as a new `<h3>`?

> Also following your idea would require translators to translate this for every single country, so it would be nice, but would also require quite a lot of effort and I worry it may be an overkill.

Yeah, that's a good point. Maybe we could handle the national championships specially and render them as "National Championship Podiums for %s", where %s becomes the country name, such as "United States" or "Poland"?

Reply:
> Sorry, I don't think I understand the problem here. Can't each thing that this person was eligible for (and won a title in!) show up as a new `<h3>`?

Sorry, I was thinking about the querying logic here, so in order for this to work we would need to
1. Get all countries the person has ever represented
2. Extend the current generic condition to explicitly check for the given country
3. Perform the query for each of the countries
4. Do the same for continents...

This just seems to be quite a lot of complexity and I am not sure if the effect would be worth it.

> Yeah, that's a good point. Maybe we could handle the national championships specially and render them as "National Championship Podiums for %s", where %s becomes the country name, such as "United States" or "Poland"?

This looks like an idea, but note that for the sake of genericness, the titles should be structured the same way, so passing a variable for national championships and not doing so for world/continental/multi-country championships seems messy.

Reply:
> This just seems to be quite a lot of complexity and I am not sure if the effect would be worth it.

I think I see what you're saying. One idea to avoid changing the queries would be to postprocess the results in Ruby, where I imagine you could write something fairly simple.

However, this sounds like it's making things more complicated, rather than simpler, so if we do want to do that, I think it's okay to handle it in a later PR. I *would* like to be consistent about the meaning of "championship type", however. Maybe invent a new word like "championship type" vs "championship level"?